### ISSUE TITLE
Android: Expandable/Collapsible State

### ISSUE THREAD

**Issue by amarlette** [facebook/react-native#30844]
## Description
- State is properly announced, but action to expand/collapse is not added. 
- Upon change state change is not announced.

## React Native version:
v0.63

**Comment by blavalla** [comment]
@amarlette , is this a duplicate of #30841 ?

**Comment by amarlette** [comment]
Yes it is I will close this issue.
